OrbitLinuxTracing/PerfEventReaders.cpp

```diff
@@ -86,7 +86,7 @@ std::unique_ptr<CallchainSamplePerfEvent> ConsumeCallchainSamplePerfEvent(
   // TODO(kuebler): we should have templated read methods
```

Comment:
Does this TODO still apply by the way, now that we pass `void*`?

Reply:
I think it does - let me see if I can address this (in a follow up PR)

Reply:
+ @florian-kuebler